packages/apps/shopify-app-remix/src/server/authenticate/admin/authenticate.ts

```diff
@@ -140,14 +140,10 @@ export function authStrategyFactory<
         shop,
       });
 
-      logger.debug('Request is valid, loaded session from session token', {
-        shop: session.shop,
-        isOnline: session.isOnline,
-      });
-
```

Comment:
This debug statement won't be printed if `strategy.authenticate` throws a `Redirect` from `afterAuth`, so I moved it to the `TokenExchangeStrategy`.